src/constants/Vocabulary.js

```diff
@@ -8,3 +8,17 @@ export const RDFS_COMMENT = "http://www.w3.org/2000/01/rdf-schema#comment";
 export const ADMIN_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/administrator";
 export const DOCTOR_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/doctor";
 export const IMPERSONATOR_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/impersonator";
+export const COMPLETE_RECORDS_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/record-complete";
+export const REJECT_RECORDS_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/reject-records";
+export const PUBLISH_RECORDS_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/publish-records";
+export const DELETE_ALL_RECORDS_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/delete-all-records";
+export const EDIT_ALL_RECORDS_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/edit-all-records";
+export const VIEW_ALL_RECORDS_TYPE = "http://onto.fel.cvut.cz/ontologies/record-manager/view-all-records";
+export const DELETE_ORGANIZATION_RECORDS_TYPE =
```

Comment:
I suggest to infer local name from url by convention:

rm_ <--- 'http://onto.fel.cvut.cz/ontologies/record-manager/'

 view_all_records <--- view-all-records

convention is that _ is used instead -

Reply:
I initially chose the (-) as the divider because I noticed in mode.ttl that other resources and properties use it, such as:
```turtle
###  http://onto.fel.cvut.cz/ontologies/record-manager/has-last-editor
rm:has-last-editor rdf:type owl:ObjectProperty ;
                   rdfs:subPropertyOf rm:relates-to .


###  http://onto.fel.cvut.cz/ontologies/record-manager/has-member
rm:has-member rdf:type owl:ObjectProperty ;
              rdfs:subPropertyOf rm:relates-to ;
              owl:inverseOf rm:is-member-of .


###  http://onto.fel.cvut.cz/ontologies/record-manager/has-owner
rm:has-owner rdf:type owl:ObjectProperty ;
             rdfs:subPropertyOf rm:relates-to .
```

However, I will change it to `_`.

Reply:
done